### DISCUSSION TITLE
I have been getting an error constantly in my repositories.

### DISCUSSION THREAD

**Discussion by TGDAL** [community/community#46304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The error it keeps on giving me is,

Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss': No such file or directory @ dir_chdir - /github/workspace/docs

Any idea?

**Comment by airtower-luna** [comment]
In the pages configuration you can choose which directory within your repository to deploy from, usually `/` (the top of the repo) or `docs/`. The error message looks like you selected `docs/` but there is no directory with that name in your repository. Changing the directory to deploy should solve the problem. :slightly_smiling_face:

**Reply by TGDAL** [reply]
How would I do that?

**Reply by airtower-luna** [reply]
See here, step 7 in particular: https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-from-a-branch
